node/follower_node_test.go

```diff
@@ -220,7 +220,7 @@ func TestSyncRoundWithRemake(t *testing.T) {
 			return true
 		}
 		return false
-	}, 10*time.Second, 500*time.Millisecond, "failed to reach newRound within the allowed time")
```

Comment:
Thanks. Did you catch this failing? I might have increased the flakiness in an attempt to reduce it. If we spot this still failing we can bump it some more, or try without a timeout. The only timeout for unit tests which I'm seeing are for `1h`, `45m`, and `30m`. If it were closer to this scale we could bump it to the max.

Reply:
I'll create a separate PR to reduce the number of rounds as well. Hopefully this will solve the problem.